resources/css/fields.css

```diff
@@ -41,4 +41,7 @@ dt-multi-select {
 dt-copy-text {
     --dt-form-padding: 0.53rem 2rem 0.53rem 0.53rem;
     display: flex;
+    --dt-text-border-radius: aliceblue;
```

Comment:
@Perumal605, Move these to [root](https://developer.mozilla.org/en-US/docs/Web/CSS/:root) so these CSS custom properties apply to all DT text fields and not just the copy-text component.

:root {
    --dt-text-border-radius: aliceblue;
    --dt-text-border-color: #d9d1d1;
}

Reply:
the styles moved to root